phoenicis-apps/pom.xml

```diff
@@ -102,13 +102,13 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
-            <version>2.7.0</version>
+            <version>2.9.0.pr3</version>
```

Comment:
This version has some conflict with some spring versions.
Honestly, I don't like the idea of not controlling which version of which dependency we are using. 
This could break the code for no reasons